.github/workflows/ci.yml

```
@@ -114,10 +114,11 @@ jobs:
         with:
           images: gresearchdev/siembol-${{ matrix.component }}
           flavor: |
-            latest=${{ github.ref == 'refs/heads/master' }}
+            latest=${{ github.ref == 'refs/heads/master' && !endsWith(steps.info.outputs.version, 'SNAPSHOT') }}
           # We use raw because our release process is not done on tag push (yet)
           tags: |
             type=raw,value=${{ steps.info.outputs.version }}
+            type=raw,value=snapshot ,enable=${{ github.ref == 'refs/heads/master' && endsWith(steps.info.outputs.version, 'SNAPSHOT') }}
```

Small space typo + same hyphen remark as above. Could you please apply the same suggestion for all `type=raw,value=snapshot` lines?
```suggestion
            type=raw,value=snapshot,enable=${{ github.ref == 'refs/heads/master' && endsWith(steps.info.outputs.version, '-SNAPSHOT') }}
```